origami/clients/rtu.py

```diff
@@ -143,12 +148,21 @@ class DeltaRejected(Exception):
 
 
 # Used in registering callback functions that get called right after squashing a Delta
-class DeltaCallback(BaseModel):
+class DeltaCallback:
     # callback function should be async and expect one argument: a FileDelta
     # Doesn't matter what it returns. Pydantic doesn't validate Callable args/return.
     delta_class: Type[FileDelta]
     fn: Callable[[FileDelta], Awaitable[None]]
 
+    def __init__(self, delta_class: Type[FileDelta], fn: Callable[[FileDelta], Awaitable[None]]):
+        # With pydantic2, raises: "TypeError: Subscripted generics cannot be used with class and instance checks"
```

Comment:
Had to comment out this well meaning check, 'cause pydantic 2 base implementation using parameterized types is incompatible with `issubclass`.

Would happily be open to suggestion on respelling it to pivot off of, say, a constant in the class or something to get the same effect?

Reply:
need to pair with you on this to better understand what's hpapening

Reply:
Sure, whenevs. If left in, the isinstance() check now raises an exception 'cause isinstance() can't be used with parameterized types.